App.js

```diff
@@ -169,7 +194,7 @@ function SettingsScreen({navigation}) {
         </Pressable>
         <Text style={titleStyle}>Settings</Text>
       </View>
-      <Pressable style={{flexDirection: 'row', marginTop: 10}} onPress={() => navigation.push('About')}>
+      <Pressable style={{flexDirection: 'row', marginTop: 10}} onPress={() => navigation.replace('About')}>
```

Comment:
Using `replace` here seems to break the navigation(as per the doc) if the user navigates from LHNHeader to `about` page.

Navigation: `LHNHeader -> Settings -> About`.

Actual:
- Pressing `back` when the user is on `about` page takes user back to chat screen.

Expected:
- Pressing `back` when the user is on `about` page takes user back to `settings` screen.

https://user-images.githubusercontent.com/35863227/214007098-5be81065-bd14-4374-bfbb-7e92cb55e84c.mov

Reply:
Why can't we keep `push` here? It doesn't seem to be breaking anything, right? and also, it makes sense because you are a pushing a new screen in the current stack.

Reply:
I think that this behaviour is expected per the design doc

Reply:
@adamgrzybowski Quick question: When you use `replace` is it meant to replace the whole nested route or just the current route? Because having `replace('Settings')` in `about` just seems to remove the `about` route and add a new route for `settings`.